### PR TITLE
Enable deletion for statefulIps.

### PR DESCRIPTION
**Pull request by hao-nan-li**
Enable deletion for statefulIps. 
https://buganizer.corp.google.com/issues/261221901

Previously the statefulIps were released with forceNew attributes because we were not able to properly delete them. Now that the Go API Library team made a fix, the stateful Ips could be properly removed.

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
compute: Enabled deletion for `statefulIps` fields in `instance_group_manager` and `region_instance_group_manager`.
```

### CODE REVIEW

**Review by zli82016 — APPROVED** [review]
Overall, LGTM. Just added a minor comment.

**Comment by zli82016** [comment]
Please wait for the review of @melinath

**Comment by hao-nan-li** [comment]
> Please wait for the review of @melinath

Meanwhile I noticed I also need to add documentation, I'll make another commit in a short.

**Review by zli82016 — APPROVED** [review]
LGTM

**Comment by zli82016** [comment]
Please wait for the review of @melinath

File: mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown [hunk 1]
```diff
@@ -160,6 +160,10 @@ group. You can specify only one value. Structure is [documented below](#nested_a
 
 * `stateful_disk` - (Optional) Disks created on the instances that will be preserved on instance delete, update, etc. Structure is [documented below](#nested_stateful_disk). For more information see the [official documentation](https://cloud.google.com/compute/docs/instance-groups/configuring-stateful-disks-in-migs).
 
+* `internal_ip` - (Optional, [Beta](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_versions.html)) Internal network IPs assigned to the instances that will be preserved on instance delete, update, etc. This map is keyed with the network interface name. Structure is [documented below](#nested_internal_ip).
```
Comment (melinath) [inline mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown:163]:
```suggestion
* `stateful_internal_ip` - (Optional, [Beta](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_versions.html)) Internal network IPs assigned to the instances that will be preserved on instance delete, update, etc. This map is keyed with the network interface name. Structure is [documented below](#nested_stateful_internal_ip).
```

File: mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown [hunk 2]
```diff
@@ -160,6 +160,10 @@ group. You can specify only one value. Structure is [documented below](#nested_a
...
+
+* `external_ip` - (Optional, [Beta](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_versions.html)) External network IPs assigned to the instances that will be preserved on instance delete, update, etc. This map is keyed with the network interface name. Structure is [documented below](#nested_external_ip).
```
Comment (melinath) [inline mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown:165]:
```suggestion
* `stateful_external_ip` - (Optional, [Beta](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_versions.html)) External network IPs assigned to the instances that will be preserved on instance delete, update, etc. This map is keyed with the network interface name. Structure is [documented below](#nested_stateful_external_ip).
```

**Review by melinath — APPROVED** [review]
LGTM apart from the docs not using the correct field names - I've noted a couple examples but they'll need to be updated throughout.

**Comment by hao-nan-li** [comment]
> LGTM apart from the docs not using the correct field names - I've noted a couple examples but they'll need to be updated throughout.

Thanks for pointing out, just made another commit to correct the field names in the doc

**Pull request merged**